kpatch-build/args.h

```diff
@@ -0,0 +1,41 @@
+#include <argp.h>
+
+extern const int NUM_ARGS;
+
+struct arguments {
+	int debug;
+	char **args;
+};
+
+extern char args_doc[];
+extern struct argp_option options[];
+extern struct argp argp;
+
+error_t parse_opt (int key, char *arg, struct argp_state *state)
```

Comment:
Yeah, I think it would be good to get rid of the whole args.h and just copy/paste where needed.